lms/security.py

```diff
@@ -31,31 +31,35 @@ class Permissions(Enum):
 class SecurityPolicy:
     """Top-level authentication policy that delegates to sub-policies."""
 
-    def __init__(self):
-        self._subpolicies = [LTISecurityPolicy(), LMSGoogleSecurityPolicy()]
```

Comment:
I think starting with `LMSGoogleSecurityPolicy` is the right move because:

- It's only used on the admin pages. It's easy to test.
- It was the only other policy that was using the older delegation mechanism in this policy (`LTISecurityPolicy` having built-int delegation itself)

Reply:
Yeah :+1: Incremental is good. Do you think we should break up `LTISecurityPolicy` into separate policies in future PRs?